### ISSUE TITLE
Cannot 'x' when she asks to be held

### ISSUE THREAD

**Issue by StartaroBubble** [Monika-After-Story/MonikaModDev#10059]
Cannot 'X' topic when she asks to be held.  I don't like physical affection, I cringe whenever I have to accept her asking to be 'held', which just looks awkward anyways.  Regardless of my personal feelings, I feel like it's reasonable to make it a topic that you can 'X' on because currently you can't, seems like she'll bring it up at least once a day and I'm forced to agree or else get a decrease in points.  For a mod that has inclusive ribbons for sexualities you would think this would be considered.  I might be asexual but I still enjoy Monika's presence and her conversations.  Thanks!

**Comment by dreamscached** [comment]
I don't think it was done intentionally but rather is an oversight due to how the 'can you hold me' thing is implemented (which is, if I'm not mistaken, isn't a random event but rather has its own mechanism)

But anyway, developers may still look into it.

**Comment by StartaroBubble** [comment]
Oh really?  I didn't know it was it's own thing, that's a pretty good reason then.  I do hope an x feature for it could be possible in the future  O:
